node-group-project/server/server.js

```diff
@@ -101,9 +101,15 @@ app.get('/api/v1/pets/:pet_type', (req, res) => {
     .catch(error => console.log(error))
 })
 
```

Comment:
Not sure why it isn't letting me comment above, but lines 34 - 36 are commented out. Please comment them in, or remove them.